### PR TITLE
Always restart docker services

### PR DESCRIPTION
**Pull request by garethbowen**
# Description

medic/cht-core#7884

# Code review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [ ] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# Compose URLs:

* [CHT Core](https://staging.dev.medicmobile.org/_couch/builds/medic:medic:7884-restart-services-always/docker-compose/cht-core.yml)
* CouchDB Single: COUCH_SINGLE_COMPOSE_URL
* [CouchDB Cluster](https://staging.dev.medicmobile.org/_couch/builds/medic:medic:7884-restart-services-always/docker-compose/cht-couchdb-clustered.yml) 
 
# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Comment by mrjones-plip** [comment]
Related to #7886

**Comment by garethbowen** [comment]
@mrjones-plip Can I get a review please? I'd like to get this into AT ASAP :)

**Review by mrjones-plip — APPROVED** [review]
tl;dr - lgtm!

---

what!? Super frustrating! I fully typed up a response and approved the PR on Friday 4th but...didn't submit??! SO sorry! I assumed this was important and spent some time on it. <sigh>  Glad I noticed an email over the weekend though!

I did have trouble testing this.  I once got API in a really good state where it was crashing a lot - restart worked well!  Otherwise I had mediocre success by restarting the container with:


`while true; do docker stop deleteme2_couchdb.2_1; echo "DOWN...sleeping for 30";sleep 30;docker start deleteme2_couchdb.2_1;echo "UP\! sleeping for 30";sleep 30; done`

And then checking restart counts with:

`docker logs deleteme2_api_1 2>&1 |egrep 'Starting API|Running server check'|tail -n5`
